backend/src/main/resources/static/css/admin/table.css

```
@@ -0,0 +1,74 @@
+
+tbody tr:hover td:first-child {
+  box-shadow: inset 3px 0 0 var(--accent);
+}
+
+/* ── Cell types ── */
+.nickname { font-weight: 700; font-size: 0.9rem; }
+
+.content-cell {
+  max-width: 320px;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.6.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)